<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editStudentWindow</class>
 <widget class="QDialog" name="editStudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit student info</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/editWindow.png</normaloff>:/icons/editWindow.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="IDArea">
       <property name="placeholderText">
        <string>Student ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkIDBtn">
       <property name="text">
        <string>check ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="deleteStudentBtn">
         <property name="text">
          <string>delete student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="changeNameOrIDBtn">
         <property name="text">
          <string>change name or ID</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="70,30">
       <item>
        <widget class="QLineEdit" name="studentNameArea">
         <property name="placeholderText">
          <string>Student name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="studentGPAArea">
         <property name="placeholderText">
          <string>student GPA</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="saveNameAndIDBtn">
       <property name="text">
        <string>save name and ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="firstYearBtn">
         <property name="text">
          <string>first year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="secondYearBtn">
         <property name="text">
          <string>second year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="thirdYearBtn">
         <property name="text">
          <string>third year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fourthYearBtn">
         <property name="text">
          <string>fourth year</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="firstSemesterBtn">
         <property name="text">
          <string>first semester</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="secondSemesterBtn">
         <property name="text">
          <string>second semester</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="70,30">
       <item>
        <widget class="QLineEdit" name="subjectNameArea">
         <property name="placeholderText">
          <string>Subject Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="subjectGradeArea">
         <property name="placeholderText">
          <string>Subject grade</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="updateSubjectBtn">
         <property name="text">
          <string>update subject</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteSubjectBtn">
         <property name="text">
          <string>delete subject</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addNewSubjectBtn">
         <property name="text">
          <string>add new subject</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QListWidget" name="subjectsArea">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
